app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt

```diff
@@ -1,19 +1,19 @@
 package com.ucmobiledevelopment.freeways.dto
 
 data class IncidentDTO(
-    val CITY: String? = "",
```

Comment:
Added an internal modifier

Reply:
This duplicates a previous review.  When that occurs, first in wins.